zipkin-server/pom.xml

```diff
@@ -283,6 +283,8 @@
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-maven-plugin</artifactId>
         <configuration>
+          <!-- This forces Spring Boot to use the PropertiesLauncher for startup, enabling loader.path -->
```

Comment:
any impact to this? especially for existing modules? cc @zeagord cc @shakuzen

Reply:
@rmichela helped me with all of the Spring stuff, so he might be able to go into more detail on this.